### ISSUE TITLE
bob.learn.em.znorm() does not work

### ISSUE THREAD

**Issue by ivana7c** [bioidiap/bob.learn.em#7]
When calling the following function, we end up with the following error:
```python
A = numpy.array([[1, 2, 3, 4, 5], [6, 7, 8, 9, 8], [7, 6, 5, 4, 3]], numpy.float64)
B = numpy.array([[5, 4, 7, 8],[9, 8, 7, 4],[5, 6, 3, 2]], numpy.float64)
normalized_scores = bob.learn.em.znorm(A, B)                

Usage (for details, see help):
ztnorm(rawscores_probes_vs_models,rawscores_zprobes_vs_models,rawscores_probes_vs_tmodels,rawscores_zprobes_vs_tmodels,mask_zprobes_vs_tmodels_istruetrial) -> output 

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: More keyword list entries (5) than format specifiers (2)
```

There is something wrong with the python bindings. In addition, the tests are wrong as well, since this problem didn't show up earlier.

It was working well on the former major release of bob.

**Comment by ivana7c** [comment]
In fact, the 'ported' unit tests look too beautiful to be true. See the following lines:
```python
  zA = znorm(my_A, my_B)
  zA_py = znorm(my_A, my_B)
  assert (abs(zA - zA_py) < 1e-7).all()
```

from:
https://github.com/bioidiap/bob.learn.em/blob/master/bob/learn/em/test/test_ztnorm.py#L116

**Comment by tiagofrepereira2012** [comment]
Fixed with the commit 61cc2540a1
Waiting the results from Travis to create a new release.
